src/uiSelectController.js

```diff
@@ -557,12 +558,12 @@ uis.controller('uiSelectCtrl',
             return false;
           }
           var inputWidth = containerWidth - input.offsetLeft - 10;
-          if (inputWidth < 50) inputWidth = containerWidth;
+          if (inputWidth < minInputWidth) inputWidth = containerWidth;
           ctrl.searchInput.css('width', inputWidth+'px');
           return true;
         };
 
-    ctrl.searchInput.css('width', '10px');
+    ctrl.searchInput.css('width', minInputWidth+'px');
```

Comment:
This method only resizes the input box where you type, which the tags aren't included in, so I don't think that should be a problem - the tags shouldn't be affected by this resizing.

But, in my testing, it seems like we have to set the input width to something small here because otherwise it will never end up on the same line as the last tag, and we will never be able to calculate how wide it should be later on. 

So this line just sets it to the smallest width we will allow the search input to be, so that when we call `updateIfVisible` later, the input will be on the same line as the last tags if possible, and we can determine what its total width should be based on that.